Assets/Scripts/Slider.cs

```diff
@@ -0,0 +1,45 @@
+using UnityEngine;
+
+
+public class Slider : MonoBehaviour {
+    [SerializeField] private float screenPart = 0.3f;
+
+    private float _savedTouchPosition;
+    private float _center;
+    private readonly float _screenWidth = Screen.width;
+
+    private float _horizontalPosition;
+
+    private float GetScreenPoint() {
+        return (_screenWidth / 2 * screenPart);
+    }
+
+    public float GetHorizontalPosition() {
+        return _horizontalPosition;
+    }
+
+    private void Update() {
+        if (Input.GetMouseButtonDown(0)) {
```

Comment:
you will have to handle win screen being open
you should be able to control the character when there is UI on top of the game